changelog/issue-2937.md

```diff
@@ -0,0 +1,5 @@
+audience: developers
+level: silent
```

Comment:
I think this kind of migration would be important for deployers to be aware of. Should wee also make the level `minor` rather than `silent`?